packages/suite/src/views/suite/SwitchDevice/DeviceItem/DeviceHeader.tsx

```diff
@@ -24,13 +27,21 @@ export const DeviceHeader = ({
     onBackButtonClick,
     isFindTrezorVisible = false,
     isDeviceStatusVisible = true,
+    actions,
 }: DeviceHeaderProps) => {
     const selectedDevice = useSelector(selectSelectedDevice);
     const isWebUsbTransport = useSelector(selectHasTransportOfType('WebUsbTransport'));
     const isDeviceConnected = selectedDevice?.connected === true;
     const deviceModelInternal = getDeviceInternalModel(device);
 
-    if (!onBackButtonClick && !onCancel && !isDeviceStatusVisible) {
+    const isDefaultCancelVisible = !actions && actions !== null && onCancel;
+
+    if (
+        !onBackButtonClick &&
+        !actions &&
+        (actions === null || !onCancel) &&
```

Comment:
Uf, this is very hard to read, I don't understand why null is not the same as undefined here